{"name": "Проверка появления номера нового заказа на экране 'Лента заказов'", "status": "broken", "statusDetails": {"message": "TypeError: 'NoneType' object is not callable", "trace": "self = <tests.test_order_page.TestOrderPage object at 0x0000010426698320>, authorize_user = <pages.main_page.MainPage object at 0x000001042669BFB0>\ncreate_order = None\n\n    @allure.title(\"Проверка появления номера нового заказа на экране 'Лента заказов'\")\n    @allure.description(\n        \"Создаём заказ и проверяем, что его номер появляется среди всех заказов на странице 'Лента заказов'\")\n    def test_order_in_order_list_success(self, authorize_user, create_order):\n        main_page = authorize_user\n    \n        # Создаём заказ\n>       create_order(main_page)\nE       TypeError: 'NoneType' object is not callable\n\ntests\\test_order_page.py:24: TypeError"}, "description": "Создаём заказ и проверяем, что его номер появляется среди всех заказов на странице 'Лента заказов'", "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1737466568113, "stop": 1737466568114, "uuid": "f0dba29f-f013-4622-99bd-b71aeed3fb20", "historyId": "81cd872b412d3beb2b9b5260aacd50e7", "testCaseId": "503030750ab22cb56dbc72d0c806c1b9", "fullName": "tests.test_order_page.TestOrderPage#test_order_in_order_list_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_page"}, {"name": "subSuite", "value": "TestOrderPage"}, {"name": "host", "value": "01ms32"}, {"name": "thread", "value": "25948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_page"}]}